### ISSUE TITLE
Does grapher support variable edge widths? 

### ISSUE THREAD

**Issue by peekxc** [ayasdi/grapher#38]
Does grapher support the ability to change the line width of edges? 

I saw there was a lineWidth option; couldn't get any results from it. What about individual line widths? It would be nice to be able to have weighted edges or something

**Comment by DianaSuvorova** [comment]
@peekxc , you should be able set line width of edges. It [defaults](https://github.com/ayasdi/grapher/blob/master/modules/renderers/renderer.js#L11) to 2

something like this should work:

```js
      var grapher = new Grapher({
        canvas: document.getElementById('graph')
        lineWidth: 10
      });

```

As for per-node line width, I don't think grapher supports it by default. You can always create your custom shader ([example](https://github.com/ayasdi/grapher/blob/master/examples/7-customShader.html)) though.

**Comment by peekxc** [comment]
@DianaSuvorova, 

I went through the code and tried doing so, but I wasn't able to get a perceptible difference on my end. Like, when I do the following with [http-server](https://github.com/indexzero/http-server): 

```bash
git clone https://github.com/ayasdi/grapher.git
cd grapher 
http-server .
...
```

And then tweak the force layout example
```javascript
// Create a grapher instance (width, height, options)
var grapher = new Grapher({
    data: network, 
    lineWidth: 5
});
```
I tried lineWidths of 1, 5, 10, and 1000, and at least for my system it doesn't appear to change anything. I am using the WebGL renderer. I'll look into creating a custom shader. I think there are nice examples lying around the internet somewhere to do that.

**Comment by DianaSuvorova** [comment]
Yes, there seems to be a problem with LINE_WIDTH  (which we [rely on](https://github.com/ayasdi/grapher/blob/master/modules/renderers/gl/renderer.js#L164) )in webGL chromium that [will never be fixed](https://bugs.chromium.org/p/angleproject/issues/detail?id=119#c17).

http://alteredqualia.com/tmp/webgl-linewidth-test/

The common workaround is to draw 2 triangles instead. We would need to update our code and  default link shader to support lineWidth option properly.

**Comment by peekxc** [comment]
Ahh, I see. I may be able to get a workaround going if I find some spare time eventually.
